debezium-ddl-parser/src/test/resources/oracle/examples/ddl_create.sql

```diff
@@ -17,6 +17,144 @@ CREATE TABLE "U3"."TEST_QUEUE_TABLE"(q_name            VARCHAR2(128),msgid
+                                                PREPROCESSOR execdir:'zcat'
+                                                LOGFILE 'deptxt1.log'
+                                                BADFILE 'deptXT.bad'
+                                                FIELDS TERMINATED BY ',' MISSING FIELD VALUES ARE NULL (recno char(2000)))
+                            LOCATION ('foo.dat.gz')) REJECT LIMIT UNLIMITED;
+
+CREATE TABLE deptxt1 ( deptno number(2), dname varchar2(14), loc varchar2(13))
+    ORGANIZATION EXTERNAL (TYPE ORACLE_LOADER DEFAULT DIRECTORY dpump_dir
+                           ACCESS PARAMETERS ( EXTERNAL VARIABLE DATA
+                                               LOGFILE 'deptxt1.log'
+                                               READSIZE=10000
+                                               PREPROCESSOR tkexcat )
+                           LOCATION ('deptxt1.dmp')) REJECT LIMIT UNLIMITED;
+
+CREATE TABLE "T_XT" ("C0" VARCHAR2(2000))
+    ORGANIZATION EXTERNAL (TYPE ORACLE_LOADER DEFAULT DIRECTORY DMPDIR
+                       ACCESS PARAMETERS ( RECORDS XMLTAG ("home address", "work address", "home phone")
+                                           READSIZE 1024
+                                           SKIP 0
+                                           FIELDS NOTRIM
+                                           MISSING FIELD VALUES ARE NULL
+                                           ( "C0" (1:2000) CHAR(2000)))
+                       LOCATION ('t.dat')) REJECT LIMIT UNLIMITED;
+
+CREATE TABLE emp_load (first_name CHAR(15), last_name CHAR(20), year_of_birth CHAR(4))
+    ORGANIZATION EXTERNAL (TYPE ORACLE_LOADER DEFAULT DIRECTORY ext_tab_dir
+                         ACCESS PARAMETERS (RECORDS DELIMITED BY '|' FIELDS TERMINATED BY ','
+                                              (first_name CHAR(7),
+                                               last_name CHAR(8),
+                                               year_of_birth CHAR(4)))
+                         LOCATION ('info.dat'));
+
+CREATE TABLE emp_load (first_name CHAR(15), last_name CHAR(20), year_of_birth CHAR(4))
+    ORGANIZATION EXTERNAL (TYPE ORACLE_LOADER DEFAULT DIRECTORY ext_tab_dir
+                         ACCESS PARAMETERS (RECORDS FIXED 20 FIELDS (first_name CHAR(7),
+                                                                    last_name CHAR(8),
+                                                                    year_of_birth CHAR(4)))
+                         LOCATION ('info.dat'));
+
+CREATE TABLE CUSTOMER_TABLE (cust_num VARCHAR2(10),
+                    order_num VARCHAR2(20),
+                    order_date DATE,
+                    item_cnt NUMBER,
+                    description VARCHAR2(100),
+                    order_total (NUMBER8,2)) ORGANIZATION EXTERNAL
+(TYPE oracle_hive
+ ACCESS PARAMETERS (
+  com.oracle.bigdata.tableName:  order_db.order_summary
```

Comment:
How should `com.oracle.bigdata.tableName` and others be defined? As it also fails 
```
line 92:32 mismatched input 'com' expecting {'BADFILE', 'BYTEORDERMARK', ...
```

Reply:
When I tested your first round of this code, I had some similar issues.  What I found was that some of the later rules in the access parameters section were causing the parser to not parse any of the elements there.  Does it fail if you comment out all the parameters except the table-name?

Reply:
No failures if I remove the `ACCESS PARAMETER` clause.